eox_nelp/edxapp_wrapper/backends/student_m_v1.py

```diff
@@ -34,3 +35,13 @@ def get_user_profile_model():
         UserProfile Model.
     """
     return UserProfile
+
+
+def get_student_api():
+    """Allow to get the student_api module from
+    https://github.com/eduNEXT/edunext-platform/blob/ednx-release/mango.master/common/djangoapps/student/models_api.py
```

Comment:
If you check the certificate code openedx changes the models_api to student_api, so that's why a used the same openedx convention, however I'm not using this anymore, this belongs to a first version, and now I'm getting the user english name from the certificate data. So I will remove this